std/process.d

```diff
@@ -1106,6 +1155,7 @@ private Pid spawnProcessPosix(scope const(char[])[] args,
                     // Until we find a way to perform this check we will try to use dlsym to
                     // check for the function. See: https://github.com/dlang/phobos/pull/9048
                     version (CRuntime_Glibc)
+                    {
```

Comment:
FYI, this confused me, because of the block declaration inside. Using version/else without braces always bugs me.